lib/mas.zsh

```diff
@@ -0,0 +1,47 @@
+mas_installed=""
+
+function mas_install_formulas() {
+  mas_installed=$(mas list 2> /dev/null)
+  for file in `dotfiles_find install.mas`; do
+    while read formula; do
+      mas_install $formula
+    done < <(grep ^ $file)
+  done
+}
+
+function mas_upgrade_formulas() {
+  outdated=$(mas outdated 2> /dev/null | cut -d ' ' -f2- | cut -d '(' -f1 | sed -e 's/ *$//' | sed -e :a -e '$!N; s/\n/, /; ta')
```

Comment:
App names can contain spaces so I can't just use `cut -d ' ' -f2`. I dunno if there's a more concise way to do this.